Adds v2 Remote Config Triggers (#1013)
firebase/functions-samples: 2nd-gen/remote-config-diff/README.md

## Get the difference between the previous and current Remote Config templates.

> With the Firebase Remote Config Cloud Function trigger you can take action when your template has been updated.

Reviewer: ```suggestion
Use the Firebase Remote Config Cloud Function trigger to take action when your template is updated.
```

> In this sample, the difference between the previous and current templates is logged.

Reviewer: ```suggestion
This sample demonstrates how to use the Remote Config Cloud Function trigger to log the difference between the previous and current Remote Config templates.
```

> See [eBay's](https://github.com/eBay/firebase-remote-config-monitor) real world implementation.

Reviewer: ```suggestion
* See [eBay's real world implementation](https://github.com/eBay/firebase-remote-config-monitor).
```

> See the [docs](https://firebase.google.com/docs/functions/rc-events) for more on Remote Config Triggers.

Reviewer: ```suggestion
* See the [docs](https://firebase.google.com/docs/functions/rc-events) for more on Remote Config Triggers.
```
